src/CosmosDB/CosmosDB/CosmosDBAccount/GetAzCosmosDBAccountKey.cs

```diff
@@ -20,9 +20,11 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.CosmosDB.Models;
 using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.CosmosDB
 {
+    [GenericBreakingChange("Output type for -Type ConnectionStrings will be changed to List<DatabaseAccountConnectionString> in next major version.")]
```

Comment:
'GenericBreakingChangeAttribute' is obsolete: 'This attribute is deprecated. Please use GenericBreakingChangeWithVersionAttribute instead